### ISSUE TITLE
Publish 0.4.3 to npm

### ISSUE THREAD

**Issue by adenflorian** [marcodejongh/chai-webdriverio#34]
In this github repo, it looks like the code is at version 0.4.3 (in the changelog) but the package on npm is still 0.4.2. Could someone publish 0.4.3 to npm? Thanks!

https://www.npmjs.com/package/chai-webdriverio

**Comment by mltsy** [comment]
I'm hoping to get this other ticket resolved also before publishing 0.4.3: https://github.com/marcodejongh/chai-webdriverio/issues/29

But both resolving that and publishing 0.4.3 are things only @marcodejongh can help with currently ;)

**Comment by adenflorian** [comment]
@marcodejongh Any news?

**Comment by mltsy** [comment]
I actually have NPM access now, so I'll publish this, I was waiting on another PR to see if they'd submit anything this weekend, but since I see no progress there, I'll just publish this! ;)

**Comment by mltsy** [comment]
Published! :checkered_flag:
